data/default_release_images.json

```diff
@@ -66,4 +66,4 @@
         "url": "quay.io/openshift-release-dev/ocp-release:4.11.0-multi",
         "version": "4.11.0-multi"
```

Comment:
Getting:
```
list versions ListSupportedOpenshiftVersions 
  get_defaults from data directory
  /assisted-service/internal/versions/versions_test.go:215
• Failure [0.178 seconds]
list versions
/assisted-service/internal/versions/versions_test.go:135
  ListSupportedOpenshiftVersions
  /assisted-service/internal/versions/versions_test.go:198
    get_defaults from data directory [It]
    /assisted-service/internal/versions/versions_test.go:215
    Expected
              
        <string>: 4.11.0-multi
    to contain substring
        <string>: 4.11.3
    /assisted-service/internal/versions/versions_test.go:251
```

Problem is, there's no multi-arch image for 4.11.3 so we can't just update this image as well. I'm trying to figure out why this image is missing